public/stylesheets/admin/style.css

```diff
@@ -0,0 +1,13 @@
+.add-book-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.add-book{
+  padding: 25px;
+  border: 1px solid black;
+  width: 60%;
```

Comment:
I'm using Skeleton as my css scaffolding. And it comes with default styling. If i don't do this the form will take up the entire window size.